jetty9/src/com/thoughtworks/go/server/util/GoPlainSocketConnector.java

```diff
@@ -0,0 +1,49 @@
+/*************************GO-LICENSE-START*********************************
+ * Copyright 2014 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *************************GO-LICENSE-END***********************************/
+
+package com.thoughtworks.go.server.util;
+
+import com.thoughtworks.go.server.Jetty9Server;
+import com.thoughtworks.go.util.SystemEnvironment;
+import org.eclipse.jetty.server.*;
+
+import static com.thoughtworks.go.util.ExceptionUtils.bomb;
+
+public class GoPlainSocketConnector implements GoSocketConnector {
+    private final int plainPort;
+    private static final int MAX_IDLE_TIME = 30000;
```

Comment:
Do you mind configuring this (idle time) in SystemEnvironment? Because, the Jetty 6 implementation earlier had no default time specified. So, it must have been using a default time. I don't know if that is 30000. Do you? If you do, then it's ok.